rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -538,6 +538,20 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 					xemu_settings->SetSetting(render->type, sstr(render->old_adapter));
 				}
 			}
+
+			std::string setting = xemu_settings->GetSetting(emu_settings::MSAA);
```

Comment:
This section feels bad. I need to test it first.

Reply:
I kinda feel like there's a better way to implement it, but I coudn't figure out how.

Reply:
```suggestion
			// Disable the AA combo box if not supported by the chosen renderer
			ui->antiAliasing->setEnabled(renderer.has_msaa);
			ui->antiAliasing->blockSignals(true);
			ui->antiAliasing->setCurrentText(renderer.has_msaa ? qstr(xemu_settings->GetSetting(emu_settings::MSAA)) : tr("Disabled"));
			ui->antiAliasing->blockSignals(false);
```

Reply:
Thanks, that's much better